setup.py

```diff
@@ -41,6 +43,9 @@
         "avalanche": [
             "avalanche @ git+https://github.com/ContinualAI/avalanche.git#egg=avalanche"
         ],
+        # TODO: (@lebrice) need to figure out a way to install cn-dpm correctly through pip, in
+        # editable mode.
```

Comment:
One thing: I'm not entirely sure about the best way to allow people to install this cn_dpm "plugin":
- Having it as a submodule in Sequoia, and install it optionally like this with extras_require (which I'm not 100% sure would work)
- Having it as an external package, with its own setup.py (as is currently the case I think) and specify a "Method" entry_point, so that Sequoia can load it as a plugin, rather than it being included within Sequoia

In any case, you can leave this as-is, and we'll fix it later.